core/eolearn/tests/test_utils/test_testing.py

```diff
@@ -0,0 +1,60 @@
+import datetime as dt
+
+import pytest
+
+from sentinelhub import CRS, BBox
+
+from eolearn.core.constants import FeatureType
+from eolearn.core.types import FeatureSpec, FeaturesSpecification
+from eolearn.core.utils.testing import patch_generator
+
+
+def test_patch_generator_default() -> None:
+    patch = patch_generator()
+    assert hasattr(patch, "bbox")
+    assert hasattr(patch, "timestamps")
```

Comment:
this can never be false, an EOPatch ALWAYS has those attributes by construction
what is the purpose of this test?

Reply:
removed